src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -138,46 +127,50 @@ protected function createWebsiteClient()
 
     /**
      * Initialize / reset the Sulu PHPCR environment.
-     *
-     * NOTE: We could use the document initializer here rather than manually creating
-     *       the webspace nodes, but it currently adds more overhead and offers
-     *       no control over *which* webspaces are created, see
-     *       https://github.com/sulu-io/sulu/pull/2063 for a solution.
      */
-    protected function initPhpcr()
+    protected function initPhpcr(HttpKernelInterface $kernel = null)
     {
+        $container = $kernel ? $kernel->getContainer() : $this->getContainer();
+
         /** @var SessionInterface $session */
-        $session = $this->getContainer()->get('doctrine_phpcr')->getConnection();
+        $registry = $container->get('doctrine_phpcr');
 
-        if ($session->nodeExists('/cmf')) {
-            NodeHelper::purgeWorkspace($session);
-            $session->save();
+        foreach ($registry->getConnections() as $session) {
+            try {
+                NodeHelper::purgeWorkspace($session);
+                $session->save();
+            } catch (NoSuchWorkspaceException $e) {
+            }
         }
 
         if (!$this->importer) {
-            $this->importer = new PHPCRImporter($session);
+            $this->importer = new PHPCRImporter($this->getContainer()->get('doctrine_phpcr')->getConnection());
         }
 
+        $connections = $registry->getConnections();
+        $connectionsKey = serialize(array_keys($connections));
```

Comment:
No .. we can use different kernel configurations with different connection configurations, for example: only the Synchronize manager tests need two document managers.